config/opensearch_dashboards.yml

```diff
@@ -23,7 +23,7 @@
 #server.name: "your-hostname"
 
 # The URLs of the OpenSearch instances to use for all your queries.
-#opensearch.hosts: ["http://localhost:9200"]
+# opensearch.hosts: ["http://localhost:9200"]
```

Comment:
I don't think we should add a space here as the rest of the configs have no space in front of them.